### ISSUE TITLE
branches with more than 2 slashes are truncated

### ISSUE THREAD

**Issue by itishermann** [Serhioromano/vscode-gitflow#38]
hi,
when i have a branch with 2 slashes in its name, the extension truncates the name 
For example:
`feature/something/something-else/another-thing`gets truncated and it shows `feature/something`

<img width="307" alt="Capture d’écran 2022-07-15 à 09 30 57" src="https://user-images.githubusercontent.com/38349706/179174769-48ee0c98-85ad-489d-b6b5-8c05a6d149e4.png">

**Comment by Serhioromano** [comment]
А что это все должно в папки превратиться? Или просто имя длинное с слешами?

**Comment by Serhioromano** [comment]
Recursion then. Let's see what I can do.

**Comment by Dragane** [comment]
Yeah same issue here. We use multiple slashes in our company as well for the branches names. Any update on this?

**Comment by sonymaker** [comment]
I also found the same problem, and I'd like to request a quick fix.

**Comment by R0bles** [comment]
Great idea! I'd actually say this is more of a bug than an enhancement — if you open a repo that uses multi-level branch names, the tree shows duplicate entries with the same name and the git flow commands stop working properly.

I put together a PR with my proposed fix ( #84 ). Tested it locally and it seems to work well, and it supports any number of nesting levels.

Example screenshot:

<img width="377" height="415" alt="Image" src="https://github.com/user-attachments/assets/92db07b6-34fe-46f8-bbce-a585fe104112" />
